CouchbaseLite.xcodeproj/project.pbxproj

```diff
@@ -3517,7 +3517,6 @@
 					};
 					27452BB71AB1180800327776 = {
 						CreatedOnToolsVersion = 6.3;
-						DevelopmentTeam = N2Q372V7W2;
```

Comment:
@lordkev hmm, on further thought, maybe this is just xcode moving stuff around ...